src/modules/dashboard/components/ActionsPage/ActionsPage.tsx

```diff
@@ -255,18 +263,35 @@ const ActionsPage = () => {
                * Typecasting it doesn't work as well
                */
               status={typeof status === 'number' && STATUS_MAP[status]}
-              createdAt={createdAt || Date.now()}
+              createdAt={createdAt}
             />
           )}
           {!!events?.length && (
-            <ul>
-              <b>Events for the tx:</b>
-              {events.map(({ name, topic }) => (
-                <li key={topic || ''}>
-                  {name} - {topic}
-                </li>
-              ))}
-            </ul>
+            <>
+              <ul>
+                <b>Events for thetx:</b>
+                {events.slice(1).map(({ name, topic }) => (
+                  <li key={topic || ''}>
```

Comment:
Why is the key here `|| ''`? Would this not make the key unique if 2 events have a falsy `topic`?

Reply:
This is just a dev thing I've added to see the topics and events on the page.

You can consider this work in progress, as I'll change both this and the types themselves when I wire it up with the UI from #2308 

You can safely ignore it, as this will not be merged until I change this.